src/libs/IOUUtils.js

```diff
@@ -67,6 +67,30 @@ function updateIOUOwnerAndTotal(iouReport, actorAccountID, amount, currency, isD
     return iouReportUpdate;
 }
 
+/**
+ * Edits the IOU total when a transaction is edited
+ *
+ * @param {Object} iouReport
+ * @param {Number} actorAccountID
+ * @param {Object} initialTransaction
+ * @param {Object} updatedTransaction
+ * @param {Boolean} isExpenseReport
+ * @returns {Object}
+ */
+function editIOUTotal(iouReport, actorAccountID, initialTransaction, updatedTransaction, isExpenseReport = false) {
```

Comment:
I am a bit concerned about the name `editIOUTotal`. We're already using `updateIOUOwnerAndTotal`.

Should this be `updateIOUTotalWithTransaction `